test/ports_test.dart

```diff
@@ -22,259 +22,335 @@ void main() {
 
 void testSingleCallbackPort() {
   test('Value', () {
-    var completer = Completer.sync();
-    var p = singleCallbackPort(completer.complete);
+    final completer = Completer.sync();
```

Comment:
Use `var` instead of `final`.
(Or don't bother about it now, I can change them in a second pass if I really want to.)

Reply:
NVM, I'll address suggestions as much as I can.